### PR TITLE
fixed: password field issue

### PR DESCRIPTION
**Pull request by vrun1208**
fixed password and confirmPassword field in register-page and password field in login-page.
field type set to password, additionally implemented toggle function to show/hide password for better user experience.
#30

### CODE REVIEW

File: frontend/src/pages/auth/Login.jsx [hunk 1]
```diff
@@ -11,12 +11,22 @@ import { isBackDropLoaderDisplayed } from "src/redux/boolean/booleanSlice";
...
 const Login = () => {
   const { handleSubmit, initialValues, isLoading, setFormValues } = useLogin();
 
   const dispatch = useDispatch();
 
+  const [toggle, setToggle] = useState(false);
+  const handleToggle = () => {
+    setToggle(!toggle);
+  }
+
```
Comment (NidhiSharma63) [inline frontend/src/pages/auth/Login.jsx]: @vrun1208 please move this logic into uselogin hook. we try to don't write logic inside the component. I'm trying to maintain clean code.

File: frontend/src/pages/auth/Register.jsx [hunk 2]
```diff
@@ -11,17 +11,38 @@ import { useDispatch } from "react-redux";
...
+  });
+  const handleToggle = (name) => {
+    setToggle({
+      ...toggle,
+      [name]: !toggle[name],
+    });
+  };
+
+  
+
```
Comment (NidhiSharma63) [inline frontend/src/pages/auth/Register.jsx]: same for this move logic into useRegister hook

**Comment by NidhiSharma63** [comment]
@vrun1208 please resolve above comments and i'm  very thankfully for your efforts

File: frontend/src/pages/auth/Login.jsx (hunk 1 above)
Comment (vrun1208) [reply frontend/src/pages/auth/Login.jsx]: Sure! working on it. And good @NidhiSharma63! these are best practices for making clean code.

Comment (NidhiSharma63) [reply frontend/src/pages/auth/Login.jsx]: @vrun1208 yes!

**Review by NidhiSharma63 — APPROVED** [review]

**Comment by NidhiSharma63** [comment]
@vrun1208 Thank you so much 😊

**Comment by vrun1208** [comment]
> @vrun1208 Thank you so much 😊

It's okayy @NidhiSharma63 ! The idea of this project interest me the most, will surely pull more on this.🚀🚀 
Also you can connect with me on [twitter](https://twitter.com/vrunpatodiya), [IG](https://www.instagram.com/v4ruun), [linkedin](https://www.linkedin.com/in/varunpatodiya)

**Comment by NidhiSharma63** [comment]
@vrun1208 sure waiting for you more PRs

**Pull request merged**